recipes-support/provisioning/phytec-board-config.bb

```diff
@@ -4,17 +4,22 @@ LIC_FILES_CHKSUM = "file://${COMMON_LICENSE_DIR}/MIT;md5=0835ade698e0bcf8506ecda
 
 SRC_URI = " \
     file://phytec-board-config.sh \
+    file://AGB_connagtive_EN_2021-07.txt \
 "
 
 S = "${WORKDIR}"
 
 do_install() {
     install -d ${D}${bindir}
     install -m 0500 ${S}/phytec-board-config.sh ${D}${bindir}/phytec-board-config
+
+    install -d ${D}/root
+    install -m 0444 ${S}/AGB_connagtive_EN_2021-07.txt ${D}/root/AGB_connagtive_EN_2021-07
```

Comment:
Please install the terms and conditions file to `${datadir}/${BPN}/`. The home directory of root should be kept clean.